CMakeLists.txt

```diff
@@ -113,6 +113,8 @@ else()
 	endif()
 	if(${SUNSHINE_ENABLE_DRM})
 		find_package(LIBDRM)
+		find_package(PkgConfig)
+		pkg_check_modules(LIBCAP libcap IMPORTED_TARGET)
```

Comment:
Please create FindLIBCAP.cmake inside the folder `cmake` and use `find_package(LIBCAP)`

The you can do:
```
if(${LIBDRM_FOUND} AND ${LIBCAP_FOUND})
  set(SUNSHINE_BUILD_KMS ON)
else()
  set(SUNSHINE_BUILD_KMS OFF)
endif()
```
And proceed to replace all further occurences of `LIBDRM_FOUND` in if statements with `${SUNSHINE_BUILD_KMS}`

You can look at `FindLIBDRM.cmake` as an example find_package file